packages/react-flow-editor/src/Editor/components/Connections/components/InputConnection.tsx

```diff
@@ -45,9 +52,9 @@ const defineDxDy = (inputPosition: Point, outputPosition: Point) => {
 }
 
 const InputConnection: React.FC<InputConnectionProps> = ({ inputPosition, outputPosition }) => {
-  const { connectorStyleConfig } = useEditorContext()
+  const { connectorStyleConfig, connectorsBehaviour = ConnectorsBehaviour.avoidSharpCorners } = useEditorContext()
```

Comment:
We should define default values in one place, i think that better to define here, when we pass value to context
`packages/react-flow-editor/src/Editor/Editor.tsx`